### PR TITLE
Use jingo-minify 0.6.0 release.

### PR DESCRIPTION
**Pull request by Natim**
This release incorporated @yohanboniface patches as well as @ubernostrum one on supporting signed commit.

### CODE REVIEW

File: requirements/prod.txt [hunk 1]
```diff
@@ -236,6 +233,9 @@ isodate==0.5.4 \
 jingo==0.9.0 \
     --hash=sha256:12a983286519303c6afd3da6f0a7e1859e69effb5fe46109424cdfca382bea56 \
     --hash=sha256:37de5435622d05763bc55d1a3d0a7edcf998ece9e6048bb470db431d1aa83c3a
+jingo-minify==0.6.0 \
+    --hash=md5:2c8c1a23a1c36bf4f4bd95150e75ecfb \
```
Comment (Natim) [inline requirements/prod.txt]: I have no idea how this ``--hash`` thing works so I took the md5 from pypi feel free to update with a proper hash.

**Comment by Natim** [comment]
A lot of praise to @willkg for the release.

File: requirements/prod.txt (hunk 1 above)
Comment (diox) [inline requirements/prod.txt]: That won't work, allowed hash algorithms for `--hash` are `sha256`, `sha384`, `sha512`.

We have a script that generates that for you, see http://addons-server.readthedocs.io/en/latest/topics/development/deps.html ; For just a single straightforward dependency like this without multiple packages you can just use:
`hashin jingo-minify==0.6.0 requirements/prod.txt`

Comment (Natim) [inline requirements/prod.txt]: Ok I will do that thanks

Comment (Natim) [inline requirements/prod.txt]:
```
hashin "jingo-minify==0.6.0" requirements/prod.txt 
Traceback (most recent call last):
  File "/usr/local/bin/hashin", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/hashin.py", line 331, in main
    verbose=args.verbose,
  File "/usr/local/lib/python3.5/dist-packages/hashin.py", line 91, in run
    run_single_package(spec, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/hashin.py", line 150, in run_single_package
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
```

Comment (diox) [inline requirements/prod.txt]: Are you launching that from the `addons-server` root ? It's working for me locally.

**Comment by Natim** [comment]
Apparently `git 1.8.3` doesn*t know about the -C option... Not sure why the docker image is using a so old git version.

**Comment by EnTeQuAk** [comment]
> Apparently git 1.8.3 doesn*t know about the -C option... Not sure why the docker image is using a so old git version.

It's build on CentOS 7

**Comment by EnTeQuAk** [comment]
https://github.com/mozilla/addons-server/pull/4732/commits/7dcbde3096bbb8543c5e8cd2389a1edc81a2c67b would need to be applied to `Dockerfile` too, not just the production docker file. Also let's cc @jasonthomas on this to make sure we want that flying around in production images.

**Comment by Natim** [comment]
Ok I will wait for his insight :)

**Comment by EnTeQuAk** [comment]
@jasonthomas do you have time to maybe look at https://github.com/mozilla/addons-server/pull/4732/files#diff-3254677a7917c6c01f55212f86c57fbfR16 and approve / disapprove about us installing an up-to-date git version like that? I think we might like to use a proper rpm for that, how do other ops parts do that?

**Comment by jasonthomas** [comment]
I am okay with using a up-to-date version of git. I think we only need git for jingo-minify. We used to use it to get the git rev for version.json but that is handled by circle-ci today.

I don't think we should use wandisco repo directly so I've mirrored this repo to our yum repo `https://s3-us-west-2.amazonaws.com/net-mozaws-prod-us-west-2-ops-rpmrepo-mirror/git/7/x86_64`. Can we update this PR to use this instead? We should follow the same method as we do for nodesource repo[1]. Is a copy of the yum.conf you can use [2] and gpg key [3].

[1] https://github.com/mozilla/addons-server/blob/master/docker/nodesource.repo https://github.com/mozilla/addons-server/blob/master/docker/nodesource.gpg.key
[2] https://gist.github.com/jasonthomas/cbd1ca19c11711167a57735008104947
[3] http://opensource.wandisco.com/RPM-GPG-KEY-WANdisco

**Comment by Natim** [comment]
Thanks @jasonthomas I will try to do so.

File: Dockerfile [hunk 2]
```diff
@@ -5,14 +5,19 @@ RUN touch /addons-server-centos7-container
...
 
 # This is temporary until https://bugzilla.mozilla.org/show_bug.cgi?id=1226533
 ADD docker/nodesource.repo /etc/yum.repos.d/nodesource.repo
 
-RUN yum install -y \
+# For git dependencies
+ADD docker/git.repo /etc/yum.repos.d/git.repo
+
+# Upgrade git
+yum install -y \
```
Comment (jasonthomas) [inline Dockerfile]: It looks like `RUN` was removed here, should be `RUN yum install ... `

File: Dockerfile.deploy [hunk 3]
```diff
@@ -9,10 +9,13 @@ ENV OLYMPIA_UID=9500
...
 
 ADD docker/nodesource.gpg.key /etc/pki/rpm-gpg/RPM-GPG-KEY-nodesource
+ADD docker/git.gpg.key /etc/pki/rpm-gpg/RPM-GPG-KEY-git
 ADD docker/epel.gpg.key /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
+
 ADD docker/nodesource.repo docker/epel.repo /etc/yum.repos.d/
+ADD docker/git.repo /etc/yum.repos.d/git.repo
 
-RUN yum install -y                                          \
+yum install -y                                              \
```
Comment (jasonthomas) [inline Dockerfile.deploy]: It looks like `RUN` was removed here, should be `RUN yum install ... `

**Comment by jasonthomas** [comment]
r+ once we fix the travis test failure, not sure why that is happening.

**Comment by EnTeQuAk** [comment]
I restarted the travis test, should be a immediate failure

**Comment by Natim** [comment]
Thanks all tests are now green :)

**Pull request merged**